README.md

```diff
@@ -105,14 +105,28 @@ class PostsController < ApplicationController
 end
 ```
 
-Notice the difference—we're now accepting a category name, rather than a category id. Even though you don't have an ActiveRecord field for `category_name`, because there is a key in the `post_params` hash for `category_name` it still calls the `category_name=` method. Oh hey! we created our own `category_name=` method! So convenient.
+Notice the difference—we're now accepting a category name, rather than a category id. Even though you don't have an ActiveRecord field for `category_name`, because there is a key in the `post_params` hash for `category_name` it still calls the `category_name=` method.
 
-We can change the view as well now:
+The setter method `#category_name=` is called whenever a `Post` is initialized with a `category_name` field. We can expand `Post.create(post_params)` to
```

Comment:
i would state this above - right after you show how to define the setter method. leave this explanation here as a reminder.

I would say we're effectively building a conveinence method to make our forms much easier for our uses to use. we set up a problem with a form accepting an ID, but we've failing to say that this solution specifically makes our app easier for our uses to use.